website/common/script/libs/shops.js

```diff
@@ -114,6 +114,27 @@ shops.getMarketCategories = function getMarket (user, language) {
 shops.getQuestShopCategories = function getQuestShopCategories (user, language) {
   let categories = [];
 
+  let bundleCategory = {
+    identifier: 'bundle',
+    text: i18n.t('questBundles', language),
+  };
+
+  bundleCategory.items = sortBy(values(content.bundles)
```

Comment:
I know we haven't started doing this, but I think it would be good to add a doc code with an example of this data structure. As in, what it looks like after fitlered and mapped.